jcl/src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -1,4 +1,4 @@
-/*[INCLUDE-IF (JAVA_SPEC_VERSION >= 8) & !OPENJDK_THREAD_SUPPORT]*/
+/*[INCLUDE-IF (JAVA_SPEC_VERSION >= 8) & (JAVA_SPEC_VERSION < 19)]*/
```

Comment:
This change prevents earlier Java levels adopt OpenJDK `j.l.Thread` w/ `OPENJDK_THREAD_SUPPORT` option. 
I think @tajila mentioned that only JDK19 intended to do so. In that case, for consistence, all `OPENJDK_THREAD_SUPPORT` occurrences need to be replaced accordingly.

Reply:
Currently, only jdk19 supports the `--enable-openjdk-thread-support` configure option.